aopy/data/base.py

```diff
@@ -159,17 +159,101 @@ def load_preproc_lfp_data(preproc_dir, subject, te_id, date, cached=True):
         subject (str): Subject name
         te_id (int): Block number of Task entry object 
         date (str): Date of recording
+        drive_number (int): drive number for multiple recordings. 1-based indexing.
 
     Returns:
-        dict: lfp data
+        ndarray: numpy array of lfp data from hdf
```

Comment:
can you be more specific about what will be returned in the cases where only one drive or multiple drives are present